Makefile

```diff
@@ -72,9 +72,9 @@ publish:
 
 bootstrap:
 	make clean-all
-	npm install
+	yarn
 	./node_modules/.bin/lerna bootstrap
 	make build
 	cd packages/babel-runtime; \
-	npm install; \
```

Comment:
How bout we keep `npm install` for this for now? I don't think we want a lock file in packages/babel-runtime and we will try to fix this via the other issue